Makefile

```diff
@@ -6,6 +6,10 @@ NOCOLOR:=\\033[0m
 
 .PHONY: test build docker-build docker-push sonobuoy-run sonobuoy-retrieve
 
+REGISTRY?=yzaccc
```

Comment:
These changes to the makefile were just for ease of iteration with my own repo rather than constantly just changing this file locally.